projects/dsp-ui/src/lib/action/components/message/message.component.html

```diff
@@ -25,11 +31,24 @@
 
     </mat-card-content>
 
-    <mat-card-footer *ngIf="!medium" class="message-footnote" [innerHtml]="message?.footnote"></mat-card-footer>
+    <mat-card-footer *ngIf="(size === 'long')" class="message-footnote">
+        <!-- Specific message from @Input -->
+        <p *ngIf="message.footnote">{{message.footnote}}</p>
+
+        <!-- Else: Default message -->
+        <p *ngIf="!message.footnote && !showLinks">
+            Please come back in a few minutes and try to <a class="link bolder" (click)="reload()">reload the page</a>.
+        </p>
+
+        <!-- Action: Contact DaSCH/DSP support -->
+        <a mat-button class="action" href="https://docs.dasch.swiss/community/faq" target="_blank">
```

Comment:
when you write "Contact the support team", most of users will expect an email address because clicking twice to land on Discuss - an app they do not know how to use - will lead to frustration and lost of tracks

Reply:
Yes I know. This should be a temporary solution and I wanted to update the documentation with a support page. This page doesn't exist yet. And discuss.dass.swiss will not be the support contact page. This has to improved and we have to think about a "Contact email-address". 

With the monitoring, which is at the moment only implemented on main server, the support team should get an error report as soon the system fails. This is part of status report on <https://status.dasch.swiss>

So, it doesn't make sense to have an e-mail-address and getting a lot of e-mails from users and anyway we didn't had the e-mail-address support@dasch.swiss until yesterday evening.

The part of support in case of a server error should be handled in an additional PR...